src/components/Books.js

```diff
@@ -1,17 +1,62 @@
-const Books = () => (
-  <div className="Books">
-    <h1>Bookstore CMS</h1>
-    <form className="formContainer">
-      <br />
-      <input type="text" placeholder="Write your book here" />
-      <br />
-      <br />
-      <input type="text" placeholder="Author" />
-      <br />
-      <br />
-      <button type="submit" className="btn">Add book</button>
-    </form>
-  </div>
-);
+import { useDispatch, useSelector } from 'react-redux';
+import { addBook, removeBook } from 'redux/books/booksSlice';
+import { useState } from 'react';
+
+const Books = () => {
+  const books = useSelector((state) => state.books.books);
+  const dispatch = useDispatch();
+
+  const categories = ['Fiction', 'Nonfiction'];
+
+  const [selectedCategory, setSelectedCategory] = useState(null);
+
+  const handleAddBook = () => {
+    const newBook = {
+      itemId: Math.floor(Math.random() * 100),
+      title: 'New Book',
+      author: 'John Doe',
+      category: selectedCategory || categories[0],
+    };
+
+    dispatch(addBook(newBook));
+  };
```

Comment:
- [ ] **Good job so far 👏 👏** However, It would be better to use a local state for ```title``` and ```author```, which can be updated through the `onChange` event listener 🙏 

Example:
```
  const [title, setTitle] = useState("")
  const [author, setAuthor] = useState("")
```
_Kindly, By using a *local state* to track `title` and `author`, will make our component responsive to user `input`, and also the user will be able to see the changes they are making in real-time. 😂_

Reply:
@TuyishimireEric Thanks for the detailed feedback. It will definitely be better if I incorporate your suggestion. Thanks.